src/main/java/uk/gov/hmcts/reform/sscs/service/holder/ReferenceDataServiceHolder.java

```diff
@@ -35,4 +46,12 @@ public class ReferenceDataServiceHolder {
     @Value("${sscs.serviceCode}")
     private String sscsServiceCode;
 
+    @Getter(lazy = true)
+    private final Map<String, List<String>> multipleHearingLocations = prepareMultipleHearingLocations();
+
+    @SneakyThrows
+    private Map<String, List<String>> prepareMultipleHearingLocations() {
+        return ImmutableMap.copyOf(OBJECT_MAPPER.readValue(new ClassPathResource("multipleHearingLocations.json").getInputStream(), new TypeReference<Map<String, List<String>>>(){}));
```

Comment:
As the HearingsDetailsMappingTest class mocks this method, the only way this method can currently be guaranteed to work is by trying it manually.
The alternative would be to write a unit test to check at the very least that it doesn't throw an exception.

I'm not compelled by a unit test for a one-liner, so I'm just going to ask: did you try this out to see if it works?

Reply:
Yes, I have tested this manually by calling the method it in the main class and it works as expected.